### PR TITLE
[add] breadcrumbs to dashboard

### PR DESCRIPTION
**Pull request by xand6r**
Add breadcrumbs to the dashboard of the admin page

### CODE REVIEW

File: src/components/dashboard/components/DashboardLayout.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
+    Route, Switch, useHistory, Link, withRouter
+} from 'react-router-dom';
+
 import { useDispatch } from 'react-redux';
 import SideNav from './SideNav';
-import { LOGOUT } from '../constants';
+import { LOGOUT, BREADCRUMB_NAME_MAP } from '../constants';
 import CreateElection from '../../createElection/components';
 import { components as ViewElection } from '../../viewElection';
 import { components as ViewStats } from '../../viewStats';
```
Comment (tosinamuda) [inline src/components/dashboard/components/DashboardLayout.js:14]: Something is off with the import order

Comment (tosinamuda) [reply src/components/dashboard/components/DashboardLayout.js:14]: Use this guide recommended by Ola to fix this https://medium.com/dooboolab/using-eslint-prettier-and-sort-imports-vscode-extensions-for-formatting-open-source-project-16edf317129d

File: src/components/addCandidate/sagas.js [hunk 2]
```diff
@@ -74,10 +74,10 @@ function* addCandidate(action) {
     /*
     *await ajax request to upload images and return a link
     */
-   if (!body.dateOfBirth || !picture) {
-       message.error(INCOMPLETE_FIELDS, WAIT_TIME);
-       yield put(actions.loadingaddCandidate(false));
-       return;
+    if (!body.dateOfBirth || !picture) {
```
Comment (tosinamuda) [inline src/components/addCandidate/sagas.js:77]: is body always set? what happens when body is null

**Review by tosinamuda — APPROVED** [review]

**Review by Way2nnadi — APPROVED** [review]
some pending Q's to answer and address, but code generally looks good

File: src/components/addCandidate/sagas.js (hunk 2 above)
Comment (xand6r) [reply src/components/addCandidate/sagas.js:77]: yes,body is always set, i set it with the payload of the action, it is a few lines above this. at the very least body is always set, then i just validate the body.

**Review by usmanoa — APPROVED** [review]
LGTM

**Pull request merged**